oeps/oep-0008.rst

```diff
@@ -0,0 +1,323 @@
+==================================
+OEP-8: Unit Testing Best Practices
+==================================
+
++---------------+-------------------------------------------+
+| OEP           | :doc:`OEP-8 </oeps/oep-0008>`             |
++===============+===========================================+
+| Title         | Unit Testing Best Practices               |
```

Comment:
I wonder if this should be generalized to include more than just unit testing. There is a lot of useful material in here about linting, requirement management, end-to-end testing etc. If not, maybe some of that material doesn't belong in here.